e2e/vitest.config.ts

```diff
@@ -10,7 +10,7 @@ try {
 
 export default defineConfig({
   test: {
-    include: ['src/{api,cli}/specs/*.e2e-spec.ts'],
+    include: ['src/{api,cli}/specs/album.e2e-spec.ts'],
```

Comment:
Can you revert this? You can run `npm test -- album` to only run one spec

Reply:
Oops. Thanks for the tip !